src/zenml/integrations/kubeflow/orchestrators/kubeflow_dag_runner.py

```diff
@@ -48,21 +49,26 @@
 
 from kfp import compiler, dsl, gcp
 from kubernetes import client as k8s_client
-from pydantic import BaseModel
 from tfx.dsl.compiler import compiler as tfx_compiler
 from tfx.dsl.components.base import base_component as tfx_base_component
 from tfx.dsl.components.base import base_node
 from tfx.orchestration import data_types
-from tfx.orchestration import pipeline as tfx_pipeline
 from tfx.orchestration.config import pipeline_config
 from tfx.orchestration.launcher import (
     base_component_launcher,
     in_process_component_launcher,
     kubernetes_component_launcher,
 )
-from tfx.proto.orchestration import pipeline_pb2
+from tfx.orchestration.pipeline import (
+    ROOT_PARAMETER as TFX_PIPELINE_ROOT_PARAMETER,
+)
+from tfx.orchestration.pipeline import Pipeline as TfxPipeline
+from tfx.proto.orchestration.pipeline_pb2 import IntermediateDeploymentConfig
+from tfx.proto.orchestration.pipeline_pb2 import Pipeline as PbPipeline
```

Comment:
Tiny nit: In the other file this was imported as `Pb2Pipeline`, might be nice to be consistent there